docs/user/readme.adoc

```diff
@@ -94,7 +94,8 @@ $ git clone https://github.com/rust-analyzer/rust-analyzer.git && cd rust-analyz
 $ cargo xtask install
 ----
 
-You'll need Cargo, nodejs and npm for this.
+You'll need Cargo, xtask, nodejs and npm for this.
+Cargo-xtask can be found at https://github.com/matklad/cargo-xtask
```

Comment:
The xtask binary is part of the rust-analyzer repository. It'll automatically build when calling `cargo xtask` from inside the rust-analyzer folder.

Reply:
Thanks for pointing that out. I changed it.